schemas/fws0_fwr_status.fbs

```diff
@@ -0,0 +1,24 @@
+// Schema for Kafka-to-NeXus Streamers status
+
+file_identifier "fws0";
+
+struct Statistics {
+avg : float;
+std : float;
+}
+
+table StreamerInfo {
+topic : string;
+messages : int;
+bytes : int;
```

Comment:
Could you make the name a little more specific or add a comment at the end of the line? What is `bytes` the size of, and is it cumulative?